quarkus/config-api/src/main/java/org/keycloak/config/SecurityOptions.java

```diff
@@ -18,6 +19,17 @@ public class SecurityOptions {
             .defaultValue(FipsMode.DISABLED)
             .build();
 
+    public static final Option<String> TRUSTSTORE_PATHS = new OptionBuilder<>("truststore-paths", String.class)
+            .category(OptionCategory.SECURITY)
+            .description("List of pkcs12 (.p12 or .pfx), PEM files, or directories containing those files that will be used as a global truststore.")
+            .build();
+
+    public static final Option<HostnameVerificationPolicy> HOSTNAME_VERIFICATION_POLICY = new OptionBuilder<>("hostname-verification-policy", HostnameVerificationPolicy.class)
```

Comment:
I was thinking more about the name for this option. Despite I previously proposed `hostname-verification-policy`, IMHO it implies it has something to do with the hostname config. Which is not true. What about `https-hostname-policy` or `https-hostname-verification` to better reflect it's TLS related. WDYT?

Reply:
Changed to tls-hostname-verification-policy - if you want something less verbose, perhaps we should just do tls-hostname-verifier to match the upstream naming (this could also be done under #24893)